docker.sh

```diff
@@ -6,10 +6,6 @@ cd $(dirname $0)
 IMAGE=ownmailbox-image
 CONTAINER=ownmailbox
 
-docker() {
-    sudo docker "$@"
-}
-
```

Comment:
I'm on OSX and I run docker without root. Executing `./docker.sh` then gives me an error, because the docker env variables aren't assigned and it can't find the daemon.

Why must docker be run as root?